go/pkg/router/dataplane.go

```diff
@@ -698,24 +695,47 @@ func (p *scionPacketProcessor) processPkt(rawPkt []byte,
 		}
 		return p.processOHP()
 	case scion.PathType:
+		res, err := p.processSCION()
+
+		if err != nil {
+			return processResult{}, err
+		}
+
 		address := p.scionLayer.SrcIA
-		stringAddress := address.String()
+		identifier := fmt.Sprintf("%s-%d", address.String(), res.EgressID)
+		rateLimiter, ok := p.d.SyncRateLimiters[p.ingressID]
 
-		if stringAddress == "1-ff00:0:110" {
+		if !ok {
```

Comment:
Is it ok if I initialize the rate limiter it in the function Run() under:
`read := func(ingressID uint16, rd BatchConn) {`

I don't know how to access the ingressID otherwise.

Reply:
Yes, exactly.